components/connection/ConnectionList.tsx

```diff
@@ -129,7 +169,7 @@ const Connections: NextPage = () => {
   );
 };
 
-export default Connections;
+export default ConnectionList;
 
 export async function getStaticProps({ locale }: GetServerSidePropsContext) {
```

Comment:
Data fetching methods can't be added here. FWIK it should be called from within `pages`.